third_party/move/move-model/bytecode/src/function_target_pipeline.rs

```diff
@@ -385,45 +392,54 @@ impl FunctionTargetPipeline {
         let rev_topo_order = Self::sort_in_reverse_topological_order(env, targets);
         info!("transforming bytecode");
         hook_before_pipeline(targets);
-        for (step_count, processor) in self.processors.iter().enumerate() {
-            if processor.is_single_run() {
-                processor.run(env, targets);
-            } else {
-                processor.initialize(env, targets);
-                for item in &rev_topo_order {
-                    match item {
-                        Either::Left(fid) => {
-                            let func_env = env.get_function(*fid);
-                            targets.process(&func_env, processor.as_ref(), None);
-                        },
-                        Either::Right(scc) => 'fixedpoint: loop {
-                            let scc_env: Vec<_> =
-                                scc.iter().map(|fid| env.get_function(*fid)).collect();
-                            for fid in scc {
-                                let func_env = env.get_function(*fid);
-                                targets.process(&func_env, processor.as_ref(), Some(&scc_env));
-                            }
-
-                            // check for fixedpoint in summaries
-                            for fid in scc {
+        let mut changed = true;
+        while changed {
+            // reset changed to assume processors wlll not change anything
+            changed = false;
+            for (step_count, processor) in self.processors.iter().enumerate() {
+                if processor.is_single_run() {
+                    processor.run(env, targets);
+                } else {
+                    processor.initialize(env, targets);
+                    for item in &rev_topo_order {
+                        match item {
+                            Either::Left(fid) => {
                                 let func_env = env.get_function(*fid);
-                                if func_env.is_inline() {
```

Comment:
This is a merge conflict and must stay. Looks like better revert this file anyway.

Reply:
This needs to be addressed.